### PR TITLE
Update search-concept-aggregation.md

### PR DESCRIPTION
**Pull request by ZChristine**

### CODE REVIEW

File: concepts/search-concept-aggregation.md [hunk 1]
```diff
@@ -373,7 +373,7 @@ Content-type: application/json
 
 ## Known limitations
 
-Aggregations are supported only for SharePoint or OneDrive items. They are not supported for **message** or **event**.
+Aggregations are supported only for SharePoint, OneDrive or External items. They are not supported for **message** or **event**.
```
Comment (isvargasmsft) [inline concepts/search-concept-aggregation.md:376]: The API Overview topics in the beta and v1 reference docs also have sections related to limitations. Would you be so kind to update those as well, please?

Documents:
https://docs.microsoft.com/en-us/graph/api/resources/search-api-overview?view=graph-rest-1.0#known-limitations

https://docs.microsoft.com/en-us/graph/api/resources/search-api-overview?view=graph-rest-beta#known-limitations

**Review by isvargasmsft — CHANGES_REQUESTED** [review]
Hello @ZChristine, 

Thank you very much for these updates. 

Please take a second to add the update to the API Overview topics as well. 

Kind regards, 

Isaac

File: concepts/search-concept-aggregation.md (hunk 1 above)
Comment (ZChristine) [reply concepts/search-concept-aggregation.md:376]: the limitations seems not in the above documents?
image

**Comment by ZChristine** [comment]
> Hello @ZChristine,
> 
> Thank you very much for these updates.
> 
> Please take a second to add the update to the API Overview topics as well.
> 
> Kind regards,
> 
> Isaac

could you help evaluate about "add the update to the API Overview topics "?

**Review by isvargasmsft — APPROVED** [review]

**Review by Lauragra — APPROVED** [review]

**Pull request merged**